__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const app = require("../app");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const connection = require("../db/connection");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  connection.end();
+});
+
+describe("/api/notARoute", () => {
+  test("GET 404", () => {
+    return request(app)
+      .get("/api/notARoute")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Route not found');
+      });
+  });
```

Comment:
there is some default behaviour in express that will throw a 404 error for non existent endpoint, but as you have a custom error handler for this purpose, this is a good test!